test_settings.py

```diff
@@ -73,3 +57,68 @@
     SOUTH_MIGRATION_MODULES = {
         'waffle': 'waffle.south_migrations'
     }
+
+if django.VERSION < (1,8):
+    # Old style TEMPLATE_* settings.
+    # Use jingo.
+
+    TEMPLATE_LOADERS = (
+        'jingo.Loader',
+        'django.template.loaders.filesystem.Loader',
+        'django.template.loaders.app_directories.Loader',
+    )
+
+    TEMPLATE_CONTEXT_PROCESSORS = (
+        'django.contrib.auth.context_processors.auth',
+        'django.core.context_processors.request',
+    )
+
+    JINGO_EXCLUDE_APPS = (
+        'django',
+        'waffle',
+    )
+
+else:
+    # New style TEMPLATES setting.
+    # Use django-jinja
+
+    INSTALLED_APPS += ('django_jinja', )
+
+    TEMPLATES = [
+        # Configure the Jinja2 template engine
```

Comment:
You might want to make this more specific. Something like:

```
Configure the Jinja2 template engine so that it looks at the templates we have in
`templates/jingo/`.
```